library/src/main/java/com/doomonafireball/betterpickers/radialtimepicker/RadialTimePickerDialog.java

```diff
@@ -804,7 +823,7 @@ private int getAmOrPmKeyCode(int amOrPm) {
                 if (amChar != pmChar) {
                     KeyEvent[] events = kcm.getEvents(new char[]{amChar, pmChar});
                     // There should be 4 events: a down and up for both AM and PM.
-                    if (events != null && events.length == 4) {
+                    if (events.length == 4) {
```

Comment:
As stated here http://developer.android.com/reference/android/view/KeyCharacterMap.html#getEvents(char[]) `events` could possibly be null, I think you should not remove the null check safety belt